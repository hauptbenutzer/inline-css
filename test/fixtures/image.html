<style>
    .illustration {
        width: 11px;
        height: 22px
    }
    .decorator {
        width: 100%;
        height: 33px;
    }
</style>
<html>
<body>
  <img class="illustration">
  <img class="decorator">
</body>
</html>
