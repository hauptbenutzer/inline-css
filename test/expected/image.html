
<html>
<body>
  <img class="illustration" width="11" height="22">
  <img class="decorator" width="100%" height="33">
</body>
</html>
